src/runtime/hexagon_host.cpp

```diff
@@ -656,6 +662,77 @@ WEAK int halide_hexagon_power_hvx_on(void *user_context) {
     return 0;
 }
 
+WEAK int halide_hexagon_power_hvx_on_mode(void *user_context, halide_hvx_power_mode_t mode) {
+    int result = init_hexagon_runtime(user_context);
+    if (result != 0) return result;
+
+    debug(user_context) << "halide_hexagon_power_hvx_on_mode\n";
+    if (!remote_power_hvx_on_mode) {
+        // The function is not available in this version of the
+        // runtime, this runtime always powers HVX on.
```

Comment:
This isn't really true. A runtime might have `halide_hexagon_power_hvx_on`, but not these new functions. Should this call `halide_hexagon_power_hvx_on` instead of doing nothing?